### PR TITLE
Describe how to use postgres in container

### PR DESCRIPTION
**Pull request by lukasz-kaniowski**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,22 @@ Keep track of your work search.
 
 ## Dev setup
 
+### Database
+
+App requires postgresql running locally. You can run it in docker fallowing below instructions
```
Comment (gjvis) [inline README.md]: Typo, "fallowing" should be "following"

**Comment by gjvis** [comment]
Ones all typo, otherwise 👍

**Comment by gjvis** [comment]
Lol at the typo in my typo comment. That should have read "one small typo"

**Comment by lukasz-kaniowski** [comment]
it's probably lack of coffeine

**Pull request merged**